README.md

```diff
@@ -136,6 +144,9 @@ This user script changes favicons of Jira pages (tab icon in browsers) to the
 avatar of individual Jira projects, which makes it easier to distinguish browser
 tabs of tickets in different projects.
 
+This scripts supports both Jira Server (self-hosted) and Jira Software
```

Comment:
Jira _Software_ is the "usual" (i.e., non-service desk) flavour of Jira. Exists both in Server and Cloud options.

Reply:
So it should just say:

> This scripts supports both Jira Server (self-hosted) and Jira Cloud.

I guess?

Reply:
And fix the incorrect plural `scripts` too.

Reply:
Fixed in [0b182de](https://github.com/rybak/atlassian-tweaks/pull/8/commits/0b182debf7ce273851f1e70681f21fa16154624d).